include/ergw_aaa.hrl

```diff
@@ -0,0 +1,32 @@
+%% Default reason mapping is not ideal but made to preserve the existing behaviour
+-define(DEFAULT_TERMINATION_CAUSE_MAPPING, [
+    {normal, 1},              % ?'DIAMETER_BASE_TERMINATION-CAUSE_LOGOUT'
+    {administrative, 4},      % ?'DIAMETER_BASE_TERMINATION-CAUSE_ADMINISTRATIVE'
+    {link_broken, 5},         % ?'DIAMETER_BASE_TERMINATION-CAUSE_LINK_BROKEN'
+    {upf_failure, 5},         % ?'DIAMETER_BASE_TERMINATION-CAUSE_LINK_BROKEN'
+    {session_error, 1},       % ?'DIAMETER_BASE_TERMINATION-CAUSE_LOGOUT'
+    {inactivity_timeout, 1},  % ?'DIAMETER_BASE_TERMINATION-CAUSE_LOGOUT'
+    {path_failure, 1}         % ?'DIAMETER_BASE_TERMINATION-CAUSE_LOGOUT'
+]).
+
+%% RFC: https://tools.ietf.org/html/rfc2866#section-5.10
+-define(RADIUS_DEFAULT_TERMINATION_CAUSE_MAPPING, [
```

Comment:
@RoadRunnr I not sure about that this mapping is expected for radius, please take a look into it.